dipy/segment/quickbundles.py

```diff
@@ -5,102 +5,110 @@
 from warnings import warn
 
 warn(DeprecationWarning("Module 'dipy.segment.quickbundles' is deprecated,"
-			" use module 'dipy.segment.clustering' instead"))
+                        " use module 'dipy.segment.clustering' instead"))
 
 
 class QuickBundles(object):
-    
-    def __init__(self,tracks,dist_thr=4.,pts=12):
-        """ Highly efficient trajectory clustering 
-        
+
+    def __init__(self, tracks, dist_thr=4., pts=12):
+        """ Highly efficient trajectory clustering
+
         Parameters
         -----------
         tracks : sequence of (N,3) ... (M,3) arrays,
                     trajectories (or tractography or streamlines)
-                    
-        dist_thr : float, 
+
+        dist_thr : float,
                     distance threshold in the space of the tracks
-        pts : int, 
-                number of points for simplifying the tracks 
-                       
+        pts : int,
+                number of points for simplifying the tracks
+
         Methods
         --------
         clustering() returns a dict holding with the clustering result
         virtuals() gives the virtuals (track centroids) of the clusters
-        exemplars() gives the exemplars (track medoids) of the clusters        
-        
+        exemplars() gives the exemplars (track medoids) of the clusters
+
         Citation
         ---------
-        
-        E.Garyfallidis, "Towards an accurate brain tractography", PhD thesis, 2012 
-        
+
+        E.Garyfallidis, "Towards an accurate brain tractography", PhD thesis, 2012
+
         """
-        warn(DeprecationWarning("Class 'dipy.segment.quickbundles.QuickBundles'"
-                                " is deprecated, use module "
-                                "'dipy.segment.clustering.QuickBundles'"
-                                " instead"))
+        warn(
+            DeprecationWarning(
+                "Class 'dipy.segment.quickbundles.QuickBundles'"
+                " is deprecated, use module "
+                "'dipy.segment.clustering.QuickBundles'"
+                " instead"))
 
         self.dist_thr = dist_thr
         self.pts = pts
-        if pts!=None:                        
-            self.tracksd=[downsample(track,self.pts) for track in tracks]
+        if pts is not None:
+            self.tracksd = [downsample(track, self.pts) for track in tracks]
         else:
-            self.tracksd=tracks                    
-        self.clustering=local_skeleton_clustering(self.tracksd, self.dist_thr)
-        self.virts=None
-        self.exemps=None                
-    
+            self.tracksd = tracks
+        self.clustering = local_skeleton_clustering(
```

Comment:
The indentation is still weird here. Any reason not break up the lines as follows? 

self.clustering = local_skeleton_clustering(self.tracksd,
self.dist_thr)

With the second line properly indented to match the indentation level of the first line?